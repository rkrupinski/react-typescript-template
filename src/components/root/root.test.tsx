import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Root from '.';

it('Renders root', function () {
  const component = renderer.create(
    <Root />
  );

  expect(component).toMatchSnapshot();
});
