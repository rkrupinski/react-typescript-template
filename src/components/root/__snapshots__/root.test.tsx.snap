// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders root 1`] = `
<button
  className="cHlpJQ"
  onClick={[Function]}
>
  Hello!
</button>
`;
